services/minespace-web/src/components/common/DocumentTable.js

```diff
@@ -40,27 +41,32 @@ export const DocumentTable = (props) => {
       dataIndex: props.categoryDataIndex,
       render: (text) => <div title="Category">{props.documentCategoryOptionsHash[text]}</div>,
     },
-    {
-      title: "Upload Date",
-      dataIndex: props.uploadDateIndex,
-      render: (text) => (
-        <div title="Upload Date">
-          {props.documentParent === "Information Requirements Table"
-            ? formatDateTime(text)
-            : formatDate(text) || Strings.EMPTY_FIELD}
-        </div>
-      ),
-    },
   ];
 
-  const userColumn = {
-    title: "User",
+  const uploadDateColumn = {
+    title: "Upload Date",
+    dataIndex: props.uploadDateIndex,
+    render: (text) => <div title="Upload Date">{formatDate(text) || Strings.EMPTY_FIELD}</div>,
+  };
+
+  const uploadDateTimeColumn = {
+    title: "Date/Time",
+    dataIndex: props.uploadDateIndex,
+    sorter: (a, b) => (moment(a.uploadDateIndex) > moment(b.uploadDateIndex) ? -1 : 1),
+    render: (text) => <div title="Date/Time">{formatDateTime(text) || Strings.EMPTY_FIELD}</div>,
+  };
+
+  const ImportedByColumn = {
```

Comment:
Any reason this is Capital case?

Reply:
The title is Date/Time to match with the existing column names casing: File Name, Category, Upload Date, Imported By.

Reply:
I was referring to the "ImportedByColumn" variable.

Reply:
Good catch, my bad. Did not realized.